packages/documentation-framework/templates/mdx.js

```diff
@@ -270,7 +270,7 @@ export const MDXTemplate = ({
                   {isDeprecated && (
                     <FlexItem spacer={{ default: 'spacerSm' }}>
                       <Tooltip content="Deprecated components are available for use but are no longer being maintained or enhanced.">
-                        <Button isInline component="span" variant="link">
+                        <Button isInline component="span" variant="link" hasNoPadding>
```

Comment:
For the 3 updates in this file, these are inline link span buttons and `hasNoPadding` only works on plain buttons. That said, I think these should all be plain buttons? Currently the label is technically taller than the button because of the display types between the span (display inline) and a label (display inline-flex), and that would be fixed by using a plain button.

<img width="1029" alt="Screenshot 2024-04-23 at 8 41 54 PM" src="https://github.com/patternfly/patternfly-org/assets/35148959/56efa891-24ab-4af4-8513-78628bc0bbdd">